Dockerfile

```diff
@@ -15,7 +15,11 @@ RUN \
   python3 -m venv /opt/venv/covid-data-model && \
   . /opt/venv/covid-data-model/bin/activate && \
   pip3 install -r requirements.txt
-  
+
 COPY . .
 
 RUN pip3 install .
```

Comment:
`RUN /opt/venv/covid-data-model/bin/pip3 install .`
This ensures you are installing into your venv